Apps/GatewayApi/src/Services/DependentService.cs

```diff
@@ -192,6 +193,40 @@ public RequestResult<IEnumerable<DependentModel>> GetDependents(string hdId, int
             return result;
         }
 
+        /// <inheritdoc/>
+        public RequestResult<IEnumerable<GetDependentResponse>> GetDependents(DateTime fromDateUtc, DateTime? toDateUtc, int? pageNumber = 0, int? pageSize = 5000)
+        {
+            // Page size max is 5000, default page number is 0
+            int size = pageSize is <= 5000 ? pageSize.Value : 5000;
+            int page = pageNumber ?? 0;
+
+            // Get Dependents from database
+            int offset = page * size;
+            DbResult<IEnumerable<ResourceDelegate>> dbResourceDelegates = this.resourceDelegateDelegate.Get(
+                fromDateUtc,
+                toDateUtc,
+                offset,
+                size);
+
+            if (dbResourceDelegates.Status != DbStatusCode.Read)
+            {
+                return RequestResultFactory.ServiceError<IEnumerable<GetDependentResponse>>(ErrorType.CommunicationInternal, ServiceType.Database, dbResourceDelegates.Message);
+            }
+
+            IEnumerable<GetDependentResponse> getDependentResponses =
+                dbResourceDelegates.Payload
+                    .Select(
+                        g => new GetDependentResponse
+                        {
+                            DelegateId = g.ProfileHdid,
+                            OwnerId = g.ResourceOwnerHdid,
+                            CreationDateTime = g.CreatedDateTime,
+                        })
+                    .ToList();
+
+            return RequestResultFactory.Success(getDependentResponses, getDependentResponses.Count(), page, size);
```

Comment:
I thought the TotalResponseCount was supposed to convey how many records match the criteria, not how many were returned in this paged set.

Reply:
In our other uses of this, it reflects the records being returned.  @MikeLyttle Please raise a ticket under Retro commits if you'd like to discuss this further (for general go forward pattern)